linera-views/src/test_utils/mod.rs

```diff
@@ -248,8 +248,8 @@ pub fn get_random_key_value_vec_prefix<R: RngCore>(
     }
 }
 
-/// Takes a random number generator rng, a number n and returns n random (key,value)
-/// which are all distinct with key and value are of length 8.
+/// Takes a random number generator rng, a number n and returns n random `(key, value)`
+/// which are all distinct with key and value being of length 8.
```

Comment:
How do you know they are all distinct? (other than high probabilities)
```
`rng`, `n`
```

Reply:
This is because the function `get_random_key_value_vec_prefix` checks if they are all distinct and if not iterate until they are.